lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -0,0 +1,98 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'js/financial-assistance/models/financial_assistance_model',
+            'text!js/financial-assistance/templates/financial_assessment_form.underscore',
+            'text!js/financial-assistance/templates/financial_assessment_submitted.underscore',
+            'js/student_account/views/FormView',
+            'text!templates/student_account/form_field.underscore'
+         ],
+         function(Backbone, $, _, gettext, FinancialAssistanceModel, formViewTpl, successTpl, FormView, formFieldTpl) {
+         	return FormView.extend({
+         		el: '.financial-assistance-wrapper',
+         		events: {
+                    'click .js-submit-form': 'submitForm'
+				},
+         		tpl: formViewTpl,
+         		fieldTpl: formFieldTpl,
+         		formType: 'financial-assistance',
+         		requiredStr: '',
+         		submitButton: '.js-submit-form',
+
+         		initialize: function(data) {
+					var context = data.context,
+						fields = context.fields;
+
+					// Add default option to array
+					if ( fields[0].options.length > 1 ) {
+						fields[0].options.unshift({
+							name: '- ' + gettext('Choose one') + ' -',
+							value: '',
+							default: true
+						});
+					}
+
+					// Set non-form data needed to render the View
+					this.context = {
+						dashboard_url: context.dashboard_url,
+						header_text: context.header_text,
+						platform_name: context.platform_name,
+						student_faq_url: context.student_faq_url
+					};
+
+					// Make the value accessible to this View
+					this.user_details = context.user_details;
+
+	                // Initialize the model and set user details
+	                this.model = new FinancialAssistanceModel({
+	                	url: context.submit_url
+	                });
+	                this.model.set( context.user_details );
+	                this.listenTo( this.model, 'error', this.saveError );
```

Comment:
I don't think it actually matters in this case, but do we want to / should we namespace events like this?